pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -84,15 +84,20 @@
     <RuntimeGroup Include="ol">
       <Parent>rhel</Parent>
       <Architectures>x64</Architectures>
-      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6</Versions>
+      <Versions>7;7.0;7.1;7.2;7.3;7.4;7.5;7.6;7.7;7.8;7.9;7.10</Versions>
```

Comment:
Have you seen any errors that require adding these specific `ol` versions? Are there errors building if the parent `rhel` version of an `ol` version is missing?